### PR TITLE
Issue 224/google UUID

### PR DESCRIPTION
**Pull request by semenovDL**
# Summary

Migrate to use github/google/uuid package instead of UUID package.

# How to test

go test ./...

# Issue

It addresses #224

# Notes

Notice that the Google's uuid.UUID is actually an alias of [16]byte. It might be a cause of problem if you treat it like string.

### CODE REVIEW

**Comment by maxekman** [comment]
May I ask if you did something different from #227?

The MongoDB driver and some other cases could be affected by the fact that it is not represented as a string anymore. I will look extra carefully at that.

File: examples/todomvc/handler_test.go [hunk 1]
```diff
@@ -196,6 +194,13 @@ func TestDelete(t *testing.T) {
 		t.Error("the body should be correct:", string(w.Body.Bytes()))
 	}
 
+	waiter := waiter.NewEventHandler()
```
Comment (maxekman) [inline examples/todomvc/handler_test.go]: Awesome!

**Comment by maxekman** [comment]
Nice work!

Tricky with the panic. Maybe just return a nil and let upstream detect the error?

**Comment by semenovDL** [comment]
@maxekman remove panic for the wrong AggregateID

Feel free to squash and merge when all test passes.

**Pull request merged**